src/renderer/api/endpoints/ingress.api.ts

```diff
@@ -107,6 +107,14 @@ export class Ingress extends KubeObject {
     }
     return ports.join(", ")
   }
+
+  getLoadBalancers() {
+    const { status: { loadBalancer = {ingress: []} } } = this;
```

Comment:
last nit, hopefully we can get the linter to eventually fix this.
```suggestion
    const { status: { loadBalancer = { ingress: [] } } } = this;
```

Reply:
Updated.